paddle/fluid/operators/activation_op.h

```diff
@@ -68,9 +71,10 @@ inline void ExtractActivationTensor(const framework::ExecutionContext& context,
     *Out = context.Output<framework::Tensor>("Out");
   }
 
-  PADDLE_ENFORCE(*Out != nullptr,
-                 "Cannot get output tensor Out, variable name = %s",
-                 context.OutputName("Out"));
+  PADDLE_ENFORCE_NOT_NULL(
+      *Out, platform::errors::NotFound(
+                "Cannot get output tensor Out, variable name = %s",
```

Comment:
cannot get tensor of output Variable Out?

Reply:
done